src/backend/mod.rs

```diff
@@ -38,6 +38,7 @@ pub trait Backend {
     /// Insert `n` line breaks to the terminal screen
     fn append_lines(&mut self, n: u16) -> io::Result<()> {
         // to get around the unused warning
+        #[allow(clippy::no_effect_underscore_binding)]
         let _n = n;
         Ok(())
     }
```

Comment:
```suggestion
    /// Insert `n` line breaks to the terminal screen
    #[allow(unused)]
    fn append_lines(&mut self, n: u16) -> io::Result<()> {
        Ok(())
    }
```

I don't understand why the `let _n = n` was used in the first place as this removes that warning, maybe wasn't an available feature before, or am I missing something ?

Reply:
My guess is that there is a concern that `n` in the doc comment and `_n` in the parameter list are noth the same, or that `_n` doesn't match the trait perhaps? Either way, I think probably just using _n in the parameter list is good enough.